ConceptMap/ConceptMap-Genomics-specimen-primarysample.json

```diff
@@ -0,0 +1,309 @@
+{
+  "resourceType": "ConceptMap",
+  "id": "genomics-primarysample",
+  "url": "https://fhir.nhs.uk/ConceptMap/primarysample",
+  "version": "0.0.1",
+  "name": "GenomicsPrimarySample",
+  "title": "Genomics Primary Sample",
+  "status": "draft",
+  "date": "2024-08-15T00:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ConceptMap used for the translation and recording of concepts from MDS source data(code-options) to equivalent Snomed CT codes, and FHIR spec.",
+  "purpose": "ConceptMap used for the translation and recording of concepts for the categories of primary sample types.This is intended for the element Specimen.type",
+  "copyright": "Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \\\"License\\\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \\\"AS IS\\\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.",
+  "targetUri": "http://snomed.info/sct?fhir_vs=ecl/%3C%3C%20123038009%20",
+  "group": [
+    {
+      "target": "http://snomed.info/sct",
+      "element": [
+        {
+          "code": "amniotic-fluid",
+          "display": "Amniotic Fluid",
+          "target": [
+            {
+              "code": "119373006",
+              "display": "Amniotic fluid",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "blood",
+          "display": "Blood",
+          "target": [
+            {
+              "code": "119297000",
+              "display": "Blood specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "bone-marrow-trephine",
+          "display": "Bone Marrow Trephine",
+          "target": [
+            {
+              "code": "258585008",
+              "display": "Bone marrow trephine specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "bone-marrrow-aspirate",
+          "display": "Bone Marrrow Aspirate",
+          "target": [
+            {
+              "code": "167913002",
+              "display": "Bone marrow source",
```

Comment:
Change to 396997002 | Specimen from bone marrow obtained by aspiration (specimen) |